packages/p2p/src/components/my-ads/my-ads-table.jsx

```diff
@@ -9,45 +10,49 @@ import { requestWS } from 'Utils/websocket';
 import { MyAdsLoader } from './my-ads-loader.jsx';
 import Popup from '../orders/popup.jsx';
 
-const headers = [
+const getHeaders = offered_currency => [
     { text: localize('Ad ID') },
-    { text: localize('Available') },
     { text: localize('Limits') },
-    { text: localize('Price') },
+    { text: localize('Rate (1 {{ offered_currency }})', { offered_currency }) },
     { text: localize('Payment method') },
-    { text: '' }, // empty header
+    { text: localize('Available amount') },
+    { text: '' }, // empty header for delete icon
 ];
 
 const type = {
     buy: localize('Buy'),
     sell: localize('Sell'),
 };
 
+const row_style = {
+    gridTemplateColumns: '1fr 2fr 2fr 2fr 2fr 1fr',
+};
+
 const RowComponent = React.memo(({ data, row_actions, style }) => (
     <div style={style}>
-        <Table.Row>
+        <Table.Row style={row_style}>
```

Comment:
I have refactor `Table.Row` & `Table.Cell` in this PR (https://github.com/binary-com/deriv-app/pull/970). Tell me what do you think about my approach. 😄

Reply:
@sam-binary and I have discussed that we should pass classnames rather than passing in a `style` prop or `flex` prop. I've updated the code to reflect this.